Packs/Clarizen/Integrations/ClarizenIAM/ClarizenIAM.py

```
@@ -0,0 +1,414 @@
+            url_suffix=uri,
+            json_data=date
+        )
+
+        if res:
+            res_json = res.json()
+            entities = res_json.get('entities')
+
+            if len(entities) > 0:
+                user_id = entities[0].get('id')
+                user_res = self.get_user_by_id(user_id)
+
+                if user_res:
+                    user_app_data = user_res.json()
+
+                    if user_app_data:
+                        user_id = user_app_data.get('id').replace('/User/', '')
+                        user_name = user_app_data.get('username')
+                        active = user_app_data.get('state', {}).get('id').replace('/State/', '')
+                        is_active = False if active == 'Disabled' else True
+
+                        return IAMUserAppData(user_id, user_name, is_active, user_app_data)
+        return None
+
+    def create_user(self, user_data: Dict[str, Any]) -> IAMUserAppData:
+        """ Creates a user in the application using REST API.
+
+        :type user_data: ``Dict[str, Any]``
+        :param user_data: User data in the application format
+
+        :return: An IAMUserAppData object that contains the data of the created user in the application.
+        :rtype: ``IAMUserAppData``
+        """
+        uri = '/data/objects/User'
+
+        res = self._http_request(
+            method='PUT',
+            url_suffix=uri,
+            json_data=user_data
+        )
+
+        if res:
+            res_json = res.json()
+            user_id = res_json.get('id')
+            user_res = self.get_user_by_id(user_id)
+
+            if user_res:
+                user_app_data = user_res.json()
+
+                if user_app_data:
+                    user_id = user_app_data.get('id').replace('/User/', '')
+                    user_name = user_app_data.get('username')
+                    is_active = True
+
+                    return IAMUserAppData(user_id, user_name, is_active, user_app_data)
```

please remove the if statements - if something doesn't go as we expect, it should raise an exception that needs to be handled in handle_exception(). same for the rest of the client methods.